.teamcity/src/main/kotlin/configurations/FlakyTestQuarantine.kt

```diff
@@ -35,7 +35,7 @@ class FlakyTestQuarantine(model: CIBuildModel, stage: Stage, os: Os, arch: Arch
         val extraParameters = functionalTestExtraParameters("FlakyTestQuarantine", os, arch, testCoverage.testJvmVersion.major.toString(), testCoverage.vendor.name)
         val parameters = (
             buildToolGradleParameters(true) +
-                listOf("-PflakyTests=only") +
+                listOf("-PflakyTests=only", "-x", ":docs:quickTest", "-x", ":docs:platformTest", "-x", ":distributions-integ-tests:quickTest", "-x", ":distributions-integ-tests:platformTest") +
```

Comment:
Regarding `distributions-integ-tests`: Can we somehow make sure that nobody uses `@Flaky` on those? Since that would mean that we wouldn't run those tests at all, right?

Reply:
I added an extra `! grep 'org.gradle.test.fixtures.Flaky' -r subprojects/distributions-integ-tests/src` step, PTAL.